proto/penumbra/penumbra/view/v1alpha1/view.proto

```diff
@@ -21,9 +21,6 @@ import "penumbra/crypto/tct/v1alpha1/tct.proto";
 // transaction-related actions, to request data from a view service, which is
 // responsible for synchronizing and scanning the public chain state with one or
 // more full viewing keys.
-//
-// View protocol requests optionally include the wallet id, used to
-// identify which set of data to query.
```

Comment:
If we did want to have a multi-tenant view service in the future, we could do so by putting the wallet ID in the request headers. This would be reasonably easy to implement on the Rust side. But not important for now, just noting that the option is open.